src/domain/app/AppNonProfileRoute.tsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+
+import AppPageTitleRoute from './AppPageTitleRoute';
+import { RouteProps } from 'react-router';
+
+type Props = RouteProps & {
+  pageTitle: string;
+};
+
+const AppNonProfileRoute = (props: Props) => {
```

Comment:
Do you need to have this compnent or could you just directly use `AppPageTitleRoute`?

Reply:
It works without, I'll remove it. For some reason my brain thought that I need another component for these 😄